redash/query_runner/mongodb.py

```diff
@@ -313,7 +314,7 @@ def run_query(self, query, user):
                 cursor = cursor.limit(query_data["limit"])
 
             if "count" in query_data:
-                cursor = cursor.count()
+                cursor = len(list(cursor))
```

Comment:
Based on the documentation, I think this supposed to be `count_documents`?

Reply:
Changed to get count using `count_documents`